upload_form.py

```diff
@@ -186,9 +186,34 @@ def uploaded_file(filename, fileformat=None):
     # column names can't have non-letters in them or javascript fails
     fix_bad_colnames(table)
 
+    column_units=[table[cln].unit for cln in table.colnames]
+    tab_metadata={'Author':'','ADS_ID':'','DOI or URL':'','Submitter':'',
+                  'IsObserved':False,'IsSimulated':False,
+                  'IsGalactic':False,'IsExtragalactic':False}
+    if len(table.meta) > 0:
+        for name, keyword in table.meta['keywords'].items():
```

Comment:
A more elegant way to do this is create a dictionary of accepted keywords : output keywords and use that.  e.g.:

```
mydict = {'author': 'Author',  'ads': 'ADS_ID', 'asd_id': 'ADS_ID'}
if name.lower() in mydict:
    outkey = mydict[name.lower()] 
    tab_metadata[outkey] = keyword['value']
```

You don't have to implement this, it's just an FYI of a (possibly?) better way to do this.